src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -425,4 +432,113 @@ public synchronized File getNextMixinMappings() {
 	public Set<File> getAllMixinMappings() {
 		return Collections.unmodifiableSet(mixinMappings);
 	}
+
+	public void runs(Closure<?> conf) {
```

Comment:
This should be `Action<?>` so mods using a kotlin dsl do not need to do any nasty `closureOf((T).() -> Unit)`

Also not sure if that should be a raw type here?

Reply:
The thing here is that there is no direct way to do this with an `Action<?>`. This is just a redirect to the Gradle API that backs `NamedDomainObjectContainer`. I'll see if I can find a workaround.

Reply:
did it a bit differently, closure is still there but can now also say `run(name, action)` in addition to `runs { name {...} }`

Reply:
You could also have `runs(Action<NamedDomainObjectContainer<RunConfigSettings>>)` to mirror Gradle's own object containers, which can be configured with `Action`s like that.

Reply:
I thought about that but thought that wouldn't work but apparently it does